metaflow/cli.py

```diff
@@ -553,6 +553,14 @@ def resume(obj,
     if step_to_rerun is None:
         clone_steps = set()
     else:
+        # validate step name
```

Comment:
just to confirm that this chunk has nothing to do with flow-level decorators per se, right?

Reply:
Ya, not sure why this snuck in here. I can make a separate PR for it too or can just leave it here if that is OK.